shared/common-adapters/input.native.js

```diff
@@ -171,7 +171,8 @@ class Input extends Component<void, Props, State> {
 
     const singlelineStyle = {
       ...commonInputStyle,
-      height: 28,
+      maxHeight: 28,
```

Comment:
using the const and put a comment. If you use height it can still stretch so instead we're using minHeight/maxHeight